scripts/core/EXT.rst

```diff
@@ -14,25 +18,42 @@ Objective
 =========
 
 Features which are device- or vendor-specific can be exposed as extensions.
-The list of extensions supported by the driver implementation can be queried using ${x}DriverGetExtensionProperties.
 
 There are two types of extensions defined by this specification:
 
 1. **Standard** - extensions ratified for inclusion into the current and all future versions of the specification.
 2. **Experimental** - extensions require additional experimentation and feedback from application vendors
    before ratification, therefore applications should not rely on experimental extensions in production.
 
-In addition, an implementation may choose to provide additional non-standard extensions via ${x}DriverGetExtensionFunctionAddress.
-It is the responsibility of the implementation to ensure these extensions do not conflict or interfere with any features or extensions of the core APIs.
-These extensions are not documented as part of this specification.
+In addition, an implementation may choose to provide additional non-standard extensions 
+It is the responsibility of the implementation to ensure these extensions do not conflict or interfere with any features or extensions of the standard 
+core or sysman APIs. These extensions are not documented as part of this specification.
+
+The following table summarizes the APIs to be used for discovery of different types of extensions.
+
++----------------------------------+-----------------------------+---------------------------------------+
+| API Category                     | Extension Type              | Extension Discovery API               |
++==================================+=============================+=======================================+
+| Core                             | Standard                    | ${x}DriverGetExtensionProperties      |
+|                                  +-----------------------------|                                       |
+|                                  | Experimental                |                                       |
++----------------------------------+-----------------------------+---------------------------------------+
+| Core                             | Implementation non-standard | ${x}DriverGetExtensionFunctionAddress |
++----------------------------------+-----------------------------+---------------------------------------+
+| Sysman                           | Standard                    | ${s}DriverGetExtensionProperties      |
```

Comment:
where is zesDriverGetExtensionProperties?

Reply:
yes adding that, this portion is just the changes to the "Extensions" page, wanted to get reviews going with it

Reply:
@jandres742 added now